README.rst

```diff
@@ -66,3 +65,24 @@ Code
 ====
 
 ``django-versatileimagefield`` is hosted on `github <https://github.com/WGBH/django-versatileimagefield>`_.
+
+Running Tests
+=============
+
+If you're running tests on Mac OSX you'll need libmagic installed. The recommended way to do this is with ``homebrew``:
+
+.. code-block:: bash
+
+    $ brew install libmagic
+
+You'll also need ``tox``:
+
+.. code-block:: bash
+
+    $ pip install tox
+
+To run tests:
+
+.. code-block:: bash
+
+    $ tox
```

Comment:
```suggestion
```suggestion
To run the entire django-versatileimagefield test matrix, that is, run all tests on all supported combination of versions of python, django and djangorestframework:

Simple run:

.. code-block:: bash

   $ tox

Note: tox names a possible combination as "tox environment"


To list the available tox environments run:

.. code-block:: bash

   $ tox -l

To execute all tests on a specific tox environment:

.. code-block:: bash

   $ tox -e <name-of-the-environment>

```
```